torch/utils/flop_counter.py

```diff
@@ -214,6 +233,12 @@ def convert_num_with_suffix(number, suffix):
     # Return the value and the suffix as a string
     return value + suffixes[index]
 
+def convert_to_percent_str(num, denom):
+    if denom == 0:
+        return "0%"
+    return f"{num / denom * 100:.2f}"
```

Comment:
```suggestion
    return f"{num / denom:.0%}"
```
See this [link](https://www.pythonmorsels.com/string-formatting/#:~:text=Percentages-,The%20.,put%20a%20%25%20sign%20after%20it.)